### DISCUSSION TITLE
Where is the Snap Assist style in the newest version

### DISCUSSION THREAD

**Discussion by YingDoge** [valinet/ExplorerPatcher#1777] [Q&A]
I cant find the Snap Assist style and i tried Disable quaddrants when snapping windows. Maybe i am blind could anyone please help me please thansk 

![image](https://github.com/valinet/ExplorerPatcher/assets/135021273/f0b691c8-4962-49c3-8c9f-39f71ba15d8d)

![image](https://github.com/valinet/ExplorerPatcher/assets/135021273/973695a0-3437-472a-b9c7-3c2bcb8ec40f)

**Comment by pyrates999** [comment]
You want the powertoy fancy zones: https://www.howtogeek.com/665780/all-microsofts-powertoys-for-windows-10-explained/#fancyzones-a-zone-based-window-manager

**Comment by KingSidorak** [comment]
WHERE IS IT?
I changed a setting in Explorer Patcher. The snapping is now not working as I want, and I cannot find whatever setting I changed in Process Explorer that caused it to be different.

**Comment by milestprower92** [comment]
> I cant find the Snap Assist style and i tried Disable quaddrants when snapping windows. Maybe i am blind could anyone please help me please thansk
> 
> ![image](https://private-user-images.githubusercontent.com/135021273/258439633-f0b691c8-4962-49c3-8c9f-39f71ba15d8d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MTkwNjcsIm5iZiI6MTcxMjYxODc2NywicGF0aCI6Ii8xMzUwMjEyNzMvMjU4NDM5NjMzLWYwYjY5MWM4LTQ5NjItNDljMy04YzlmLTM5ZjcxYmExNWQ4ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOFQyMzI2MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZWRjOWQ5NDk1NjIzZDFlOTE5MmNkM2I3ODUyZjM1MDk4MjU5ZDQ2YjNiMTQ4NzFlZGYzODE3MzlmN2VjODVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6RoE8CVmJ96DY9t7X2QC31DqIxWMNRuaFqBmGRq5_ss)
> 
> ![image](https://private-user-images.githubusercontent.com/135021273/258439789-973695a0-3437-472a-b9c7-3c2bcb8ec40f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MTkwNjcsIm5iZiI6MTcxMjYxODc2NywicGF0aCI6Ii8xMzUwMjEyNzMvMjU4NDM5Nzg5LTk3MzY5NWEwLTM0MzctNDcyYS1iOWM3LTNjMmJjYjhlYzQwZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOFQyMzI2MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNmYzYzcyNWM3NjQ5MjU2MzM2NDY3MzNkMDM4Y2ExZTgyODQ1MjBjYzcyZDUzMDIzMzg2MjMzYmJlZDk1YjQ2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.hkCqJmHqjqEn_uwzdp-IC8_OvYk7Dq4BXh-sjju-eFg)

Microsoft removed the windows 10 snap assist style on 22621+
